cpp/src/replace/replace.cu

```diff
@@ -142,7 +150,11 @@ namespace{ //anonymous
 
 template <typename Type>
 __global__
-void replace_nulls_with_scalar(gdf_size_type size, Type* out_data, gdf_valid_type * out_valid, const Type *in_data_scalar) 
+  void replace_nulls_with_scalar(gdf_size_type size,
+                                 const Type* in_data,
```

Comment:
Be consistent in your use of east const and west const (they are mixed here in a single function prototype).

Reply:
Done and declaring my allegiance to the one true const